test/controllers/offenses_controller_test.rb

```diff
@@ -8,18 +8,18 @@ class OffensesControllerTest < ActionController::TestCase
     @offense = offenses(:one)
   end
 
-  test "should get index" do
```

Comment:
Switching `"` to `'` per Ruby standards.  Not sure why this isn't the default...